src/coreclr/src/jit/importer.cpp

```diff
@@ -14120,9 +14120,9 @@ void Compiler::impImportBlockCode(BasicBlock* block)
 
                 lclTyp = JITtype2varType(ciType);
 
-#ifdef _TARGET_AMD64
+#ifdef TARGET_AMD64
                 noway_assert(varTypeIsIntegralOrI(lclTyp) || varTypeIsFloating(lclTyp) || lclTyp == TYP_STRUCT);
```

Comment:
@dotnet/jit-contrib @BruceForstall - this is a `noway_assert` that has apparently been inactive for at least some time. Should it simply be eliminated?

Reply:
Seems reasonable to just delete, but someone more familiar with the importer than I should make the call.

Reply:
You might be the best one to make a call on this; it apparently came in with dotnet/coreclr#2545...
```
git blame -L 14047 -- importer.cpp
05b0c2cfdf0b (Carol Eidt                                           2015-12-08 18:15:15 -0800 14047) #ifdef _TARGET_AMD64
```

Reply:
I don't think it's useful, since `JITtype2varType` will always return one of these for a field. I would vote to delete it.

Reply:
Fine by me.